bazel/glog.bzl

```diff
@@ -38,58 +78,65 @@ def glog_library(namespace = "google", with_gflags = 1, **kwargs):
             "src/utilities.cc",
             "src/utilities.h",
             "src/vlog_is_on.cc",
+        ] + select({
+            "@bazel_tools//src/conditions:windows": ["src/windows/port.cc", "src/windows/port.h"],
+            "//conditions:default": [":config_h"],
+        }),
+        copts =
+            select({
+                "@bazel_tools//src/conditions:windows": common_copts + windows_only_copts,
+                "@bazel_tools//src/conditions:darwin": common_copts + linux_or_darwin_copts + darwin_only_copts,
+                "//conditions:default": common_copts + linux_or_darwin_copts,
+            }),
+        deps = [
+            ":glog_headers",
+        ] + gflags_deps,
+        **kwargs
+    )
+
+    # glog headers vary depending on the os.
+    native.cc_library(
+        name = "glog_headers",
+        deps = select({
+            "@bazel_tools//src/conditions:windows": [":windows_glog_headers"],
+            "//conditions:default": [":default_glog_headers"],
+        }),
+    )
+
+    native.cc_library(
+        name = "windows_glog_headers",
+        hdrs = ["src/glog/log_severity.h", "src/windows/config.h",] + native.glob(["src/windows/glog/*.h"]),
```

Comment:
This will leak config.h into the include path, which may break targets that depend on glog and contain `#include "config.h"`. Can you apply the pattern from https://github.com/bazelbuild/bazel/issues/4463 to calculate the path of `src/windows/`, and add `"-I" + src_windows` to `windows_only_copts`?